{"status":"FAIL","msg":"FAIL:\n ReferenceError: message is not defined\n    at ../../test262/harness/assert.js:37\n    at anonymous (../../test262/test/language/arguments-object/10.6-6-3.js:14)\n    at anonymous (../../test262/test/language/arguments-object/10.6-6-3.js:14)\n    at testcase (../../test262/test/language/arguments-object/10.6-6-3.js:16)\n\n","path":"../../test262/test/language/arguments-object/10.6-6-3.js"}